airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/__init__.py

```diff
@@ -3,7 +3,9 @@
 #
 
 from airbyte_cdk.sources.declarative.auth.oauth import DeclarativeOauth2Authenticator
+from airbyte_cdk.sources.declarative.auth.jwt import JwtAuthenticator
 
 __all__ = [
     "DeclarativeOauth2Authenticator",
+    "JwtAuthenticator"
```

Comment:
For consistency, should this also be `DeclarativeJWTAuthenticator?`

Reply:
It seems the oauth2 auth would be the outlier: `ApiKeyAuthenticator`, `BearerAuthenticator`, `BasicHttpAuthenticator`, etc don't explicitly note declarative.